packages/odoo/src/Adaptor.js

```diff
@@ -60,33 +60,44 @@ async function login(state) {
   return state;
 }
 /**
- * Create a record in Odoo
+ * Create a record in Odoo.
```

Comment:
I note that we have no `@state` docs on create (or the other apis). Can we add them now? state.data is interesting here because it changes depending on `downloadNewRecord`